affinity/cpp-23/d1436r1.md

```diff
@@ -191,13 +199,13 @@ The initial solution proposed by this paper may only target systems with a singl
 
 ## Overview
 
-In this paper we propose an interface for discovering the execution resources within a system, querying the relative affinity metric between those execution resources, and then using those execution resources to allocate memory and execute work with affinity to the underlying hardware those execution resources represent. The interface described in this paper builds on the existing interface for executors and execution contexts defined in the executors proposal [[22]][p0443].
+In this paper we propose executor properties that can be used for querying the affinity between different hardware an software resources within a system execute and to require binding of *execution agents* to the underlying hardware or software resources in order to achieve performance through data locality. These properties provide a low granularity and is aimed at users who may have little or no knowledge of the system architecture.
```

Comment:
- "... different hardware and software resources ..."
- "... within a system execute" doesn't make sense; did you mean something like "within a system available to executors"?

Reply:
Good catch thanks, for "... within a system execute" I am not exactly sure what I was trying to say there, it may have just been a typo, but I like the wording you suggested so I'll use that.